raiden-ts/src/path/actions.ts

```diff
@@ -20,3 +25,12 @@ export const pathFindFailed = createStandardAction('pathFindFailed').map(
 export const pfsListUpdated = createStandardAction('pfsListUpdated')<{
   pfsList: readonly Address[];
 }>();
+
+export const persistIOU = createStandardAction('persistIOU')<
```

Comment:
Nitpick: can these actions be named as `${domain}{verb}` for consistency? e.g. `iouPersist` & `iouClear`